Dockerfile

```diff
@@ -20,19 +20,21 @@
 FROM buildpack-deps:bullseye AS build-env
 
 # Install dependencies
-RUN apt-get update && apt-get install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev protobuf-compiler cmake ca-certificates curl
-RUN update-ca-certificates
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev protobuf-compiler cmake curl
 
-# Install rustup
-# https://rustup.rs/
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
-WORKDIR /usr/src/forest
+WORKDIR /forest
 COPY . .
 
-# Install Forest
-RUN make install
+# Install Forest. Move it out of the cache for the prod image.
+RUN --mount=type=cache,sharing=private,target=/root/.cargo/registry \
```

Comment:
How about using `sccache`, so that we can use it for CI as well

Reply:
Not sure I get it. It's not in the Docker [documentation](https://docs.docker.com/engine/reference/builder/#mount-types). Could you please elaborate?

Reply:
I was suggesting some extra steps
In dockerfile:
1. Install sccache
2. RUSTC_WRAPPER = "sccache rustc"
3. CC = "sccache clang"
4. CXX = "sccache clang++"
5. mount ~/.cache/sccache from host
In CI manifest:
`uses: hanabi1224/sccache-action@v1.2.0`

Reply:
I am not sure about it. It seems to would complicate the otherwise relatively straightforward Dockerfile, and the cache I added here would still need to be present (rustup reasons). Perhaps we can touch this in another issue and verify that it significantly decreases the CI duration. What do you think?

Reply:
Sounds good